platform/solr/platform-solr-server-standalone/src/main/resources/solr/conf/schema_metacard.xml

```diff
@@ -234,9 +234,19 @@
         <fieldType name="text_ws" class="solr.TextField" positionIncrementGap="100">
       <analyzer>
         <tokenizer class="solr.WhitespaceTokenizerFactory"/>
+          <filter class="solr.PatternReplaceFilterFactory" pattern="^\p{Punct}*(.*?)\p{Punct}*$" replacement="$1"/>
```

Comment:
Was there anything else that was different between schema.xml and schame_metacard.xml that was not included?